### PR TITLE
CASM-4137 Improving iuf messages for management-nodes-rollout stage

### PR DESCRIPTION
**Pull request by sravani-sanigepalli**
# Description
Improve contextual logging information for management-nodes-rollout operation
Changes in this PR address the following use cases for IUF -

As an admin, I want to have more contextual information from the result of prepare-images stage so I can do deployments manually
As an admin, I want to have better logging information about what failed and what I need to fix

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: workflows/iuf/operations/management-nodes-rollout/management-two-master-nodes-rollout.yaml
Comment (JohnNavitskyHPE) [inline workflows/iuf/operations/management-nodes-rollout/management-two-master-nodes-rollout.yaml:259]: it would be nice to see an INFO message here when the result code is 0 that it successfully set the parameters for the host, including all the items it set.  like

"Successfully set cray bss bootparameters for host <host> with --kernel <kernel>, --initrd <initrd> and --params <params>" or something like that.

**Review by JohnNavitskyHPE — APPROVED** [review]

**Comment by deantroe** [comment]
I won't have time to review this due to other priorities, so don't wait for my approval.

**Review by atifsyedali — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Pull request merged**
